### ISSUE TITLE
Remplacer le pavé de texte par une vraie page d'accueil

### ISSUE THREAD

**Issue by Mozinet-fr** [mozfr/www#178]
Le pavé de texte en page d'accueil es indigeste et ferait fuir le plus motivé des libristes. On pourrait avoir à la place soit une page web unique avec des sections ou une sorte de hub qui permette de renvoyer à toutes les sous-pages et aux liens et outils que l'on peut chercher en allant sur la page d'accueil de Mozilla fr.

**Comment by pascalchevrel** [comment]
Je suis d'accord et j'avais eu la même idée, par contre j'ai pas trop le temps de bosser là dessus (le contenu surtout)

**Comment by Mozinet-fr** [comment]
On essaie de regarder avec les autres de la comm ce week-end pour faire se faire une première idée d'une possible maquette.
